test/extensions/filters/listener/proxy_protocol/proxy_protocol_test.cc

```diff
@@ -345,6 +345,13 @@ TEST_P(ProxyProtocolTest, ErrorRecv_2) {
   EXPECT_CALL(os_sys_calls, close(_)).Times(AnyNumber()).WillRepeatedly(Invoke([this](os_fd_t fd) {
     return os_sys_calls_actual_.close(fd);
   }));
+#ifdef WIN32
```

Comment:
This doesn't seem ideal, can we avoid special casing Windows in test that aren't platform specific?

Reply:
Sure, let me see what is the best way to go about removing the `#ifdef`

Reply:
Thanks for the review, fixed in 9319ca061d5ecdc64f3a9c135393f59d62e5138b